src/ReactWrapper.jsx

```diff
@@ -885,12 +885,23 @@ class ReactWrapper {
   }
 
   /**
-   * Returns true if the current wrapper has no nodes. False otherwise.
+   * Delegates to exists()
    *
    * @returns {boolean}
    */
   isEmpty() {
-    return this.length === 0;
+    // eslint-disable-next-line no-console
+    console.warn('Deprecated method isEmpty() called, use exists() instead.');
```

Comment:
I think it would be nice to prefix this with `Enzyme::` just so people know where the warning is coming from.